dashboard/containerAuthentication.md

```diff
@@ -0,0 +1,49 @@
+[Home](../README.md) | [Dashboard](dashboard.md) | [Container Authentication](containerAuthentication.md)
+
+# Azure Active Directory App Registration Setup
+
+## Purpose
+
+This guide provides a quick setup for registering an Azure Active Directory (AAD) application. It includes creating a client secret, setting up API permissions, and preparing variables for pipeline use. This will allow for containers to authenticate the requests tht are being received.
```

Comment:
```suggestion
This guide provides a quick setup for registering an Azure Active Directory (AAD) application. It includes creating a client secret, setting up API permissions, and preparing variables for pipeline use. This will allow for containers to authenticate the requests that are being received.
```